src/Components/ComparePublicationData/ComparePublicationDataModal.tsx

```diff
@@ -497,6 +491,7 @@ const ComparePublicationDataModal: FC<ComparePublicationDataModalProps> = ({
                   <Grid container spacing={2} justifyContent="flex-end" alignItems="baseline">
                     <Grid item>
                       <Button
+                        disabled={isLoadingContributors}
```

Comment:
Quick and dirty løsning istedenfor å legge inn bøttevis av arrays av abort-controllere for api-kallene som burde vært avbrutt. For de fleste poster (ikke monsterposter) spiller det ikke noen rolle da bidragsyterne lastes inn kjapt .